core/templates/dev/head/pages/exploration-player-page/learner-experience/tutor-card.directive.html

```diff
@@ -25,7 +25,7 @@
           ng-click="toggleShowPreviousResponses()">
         <span class="conversation-skin-responses-dropdown-text">
           <span translate="I18N_PLAYER_PREVIOUS_RESPONSES"
-                translate-values="{previousResponses: <[getDisplayedCard().getInputResponsePairs().length - (isInteractionInline() ? 1 : 0)]>}">
+                translate-values="{previousResponses: getDisplayedCard().getInputResponsePairs().length - (isInteractionInline() ? 1 : 0)}">
```

Comment:
To test:
- Create an exploration with interaction.
- Go to preview mode.
- Add a few responses.
- Check the "Previous response (<int>)" message.